### ISSUE TITLE
Make extension available in Firefox for Android

### ISSUE THREAD

**Issue by RealStickman** [floccusaddon/floccus#1485]
### Describe the feature you'd like to request

Firefox is now allowing all extensions marked as "Android compatible" on mobile.  
This would mean having Floccus directly in the browser like the desktop versions currently.

### Describe the solution you'd like

[Mozilla news piece about extension support on Android](https://blog.mozilla.org/addons/2023/11/28/open-extensions-on-firefox-for-android-debut-december-14-but-you-can-get-a-sneak-peek-today/)  

[Porting guide](https://extensionworkshop.com/documentation/develop/developing-extensions-for-firefox-for-android/)

### Describe alternatives you've considered

The current application works fine, though I often forget to use it and it is slightly less convenient.

**Comment by marcelklehr** [comment]
Yes, good idea. Unfortunately, firefox for Android does not support all extensions, yet. Floccus is not among the supported

**Comment by mvevitsis** [comment]
I sideloaded it on Firefox beta, however it does not work.

Importing a profile does nothing.

I believe it is related to this:
https://bugzilla.mozilla.org/show_bug.cgi?id=1812408

**Comment by marcelklehr** [comment]
Once the "Firefox for Android" column on this page turns green, we can talk again :)

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/bookmarks#browser_compatibility

**Comment by hrehfeld** [comment]
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs#browser_compatibility

tabs api has a bunch of features supported already. Can you support tab syncing only at first? :dollar:

**Comment by drove5065** [comment]
@marcelklehr can floccus provide a minimum support on android firefox, by simply saving only the the current tab into favorites?

**Comment by marcelklehr** [comment]
> Can you support tab syncing only at first?

That's an interesting angle, I will look into this, yes!
